pkg/proxy/userspace/proxier_test.go

```diff
@@ -225,6 +225,15 @@ func waitForNumProxyClients(t *testing.T, s *ServiceInfo, want int, timeout time
 	t.Errorf("expected %d ProxyClients live, got %d", want, got)
 }
 
+func startProxier(p *Proxier, t *testing.T) {
```

Comment:
This is adding a `SyncLoop` thread and calls to `OnServiceSynced` and `OnEndpointsSynced`, but that has nothing to do with the cleanup/shutdown changes. It just snuck in here from a later commit.